source/darwin/darwin_pki_utils.c

```diff
@@ -66,7 +66,6 @@ int aws_import_ecc_key_into_keychain(
         AWS_ZERO_STRUCT(import_params);
         import_params.version = SEC_KEY_IMPORT_EXPORT_PARAMS_VERSION;
         import_params.passphrase = CFSTR("");
-        format = kSecFormatUnknown;
```

Comment:
A few lines up this is initialized as `SecExternalFormat format = kSecFormatOpenSSL;` each time we loop.

That's on purpose, right? 

It seems OK to me, but the PR description "we need to reset it to unknown before every call", but maybe the PR description is describing the NEXT change?

Reply:
yes, that was an error in my previous pr. we dont need unknown format here, since it already specifies a format couple lines above.